docs/zkapps/faq.mdx

```diff
@@ -141,7 +142,11 @@ Use the `requireSignature` command. See the [requireSignature](/zkapps/o1js-refe
 
 ### How do I configure who has the authority to interact and make changes to a specific part of an account?
 
+<<<<<<< HEAD
```

Comment:
this looks like a unresolved merge conflict

Reply:
fixed the missed merge conflicts - found 5